src/components/ui/msig-proposal.vue

```diff
@@ -523,89 +475,13 @@ export default {
         this.transactionCallback('e_unapproval')
       }
     },
-    // deny a proposal, just marks as an action
-    async denyProposal (proposer, proposalName) {
```

Comment:
Seeing that you removed this makes me think we might need a `denyproposal` action on the msig contract. it doesn't necessarily need to have a logical impact on the state of a msig but it should an existing approval if there is one. I remember we added the denyproposal action to signal that custodians had seen the proposal and deliberately chosen to deny - which is different than simply ignoring a prop.

Reply:
Okay, I'll add a separate task for this. The buttons were commented out in the UI, so it looked like it's no longer in use. I suggest we merge this as is and have this change in a separate branch.